src/main/java/org/jabref/gui/entryeditor/fileannotationtab/FileAnnotationTabView.java

```diff
@@ -94,12 +94,14 @@ private Node createFileAnnotationNode(FileAnnotationViewModel annotation) {
         Label date = new Label(annotation.getDate());
         Label page = new Label(Localization.lang("Page") + ": " + annotation.getPage());
 
-        marking.setStyle("-fx-font-weight: bold");
+        marking.setStyle("-fx-font-size: 10px; -fx-font-weight: bold");
```

Comment:
Please specify the font-size in `em` so that it scales properly if the base font size is changed (e.g. on high res displays).

Reply:
Changed to `0.75em`